tests/unit/test_pkgx_load_urls.py

```diff
@@ -26,6 +26,8 @@ def test_scenarios():
     Collection of test scenarios for URL loading.
     Each scenario represents a different case we want to test.
     """
+    # TODO; add a scenario to check if there's no URLs in the db, but the transformer
+    # found some
```

Comment:
https://coverage.readthedocs.io/en/7.8.1/ can help you find what you're _not_ testing (if you're not already using it).